mesoscope/cellpack
# Feature/upload local recipes to s3

Problem
=======
What is the problem this work solves, including
Partially closing #140 

Solution
========
What I/we did to solve this problem
- Updated `simularium_helper.py` to:
      - store local results to S3 path: `cellpack-results/simularium` 
      - enable automatically opened packing results on Simularium
- Updated README 

Note: a separate PR will handle the upload of metadata to firebase 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Pack a local recipe: `pack -r cellpack/tests/recipes/v2/test_spheres.json -c examples/packing-configs/sparse-packing.json`
2. Check the uploaded object in S3 bucket, should look like this:

<img width="1331" alt="Screenshot 2023-08-29 at 3 00 31 PM" src="https://github.com/mesoscope/cellpack/assets/91452427/d76e6532-bb01-4aa4-8c71-bdfd731a1886">


Keyfiles (delete if not relevant):
-----------------------
1. `simularium_helper.py`
2. `AWSHandler.py`
